src/msg.hpp

```diff
@@ -196,13 +196,13 @@ class msg_t
         // zero-copy LMSG message for v2_decoder
         type_zclmsg = 105,
 
+        type_max = 105,
+
         //  Join message for radio_dish
         type_join = 106,
 
         //  Leave message for radio_dish
-        type_leave = 107,
-
-        type_max = 107
+        type_leave = 107
```

Comment:
not sure I follow the reason for this change?

Reply:
Otherwise we get an error when we close the msg because of https://github.com/zeromq/libzmq/blob/master/src/msg.cpp#L50 and https://github.com/zeromq/libzmq/blob/master/src/msg.cpp#L218.

Reply:
I think this was meant to prevent leaking private msg types.

Reply:
it's to check that the type is in range and wasn't corrupted - this will make the check fail for a join/leave message

Reply:
Right, I'm blind.